src/cueUtilities.js

```diff
@@ -319,60 +320,113 @@ module.exports = function (params, cy, api) {
+          const expandcollapseRenderedEndX = expandcollapseRenderedStartX + expandcollapseRenderedRectSize;
+          const expandcollapseRenderedEndY = expandcollapseRenderedStartY + expandcollapseRenderedRectSize;
+
+          let posX;
+          let posY;
+          if (event instanceof MouseEvent) {
+            posX = event.offsetX;
+            posY = event.offsetY;
+          } else if (event instanceof TouchEvent) {
+            // needed so it matches the range for expandcollapses rendered locations
+            const canvasRect = event.target.getBoundingClientRect();
+            const touchPageCoordinates = getTouchPageCoordinates(event);
+            posX = touchPageCoordinates.x - canvasRect.x;
+            posY = touchPageCoordinates.y - canvasRect.y;
+          } else {
+            return false;
+          }
+
+          return posX >= expandcollapseRenderedStartX && posX <= expandcollapseRenderedEndX &&
+                  posY >= expandcollapseRenderedStartY && posY <= expandcollapseRenderedEndY;
+        }
+
+        function getGroupOfTappedOnCue(event) {
+          const clickedGroup = selectedGroups.find(group => isEventOnCueRegion(group, event));
+          return clickedGroup ? clickedGroup : null;
+        }
+
+        function cueTap(event) {
+          let node;
+          if (hoveredGroup && isEventOnCueRegion(hoveredGroup, event)) {
+            node = hoveredGroup;
+          }
+          else if (selectedGroups.length > 0) {
+            node = getGroupOfTappedOnCue(event);
+          }
+
+          if (!node) {
+            return;
+          }
+          let opts = options();
+
+          event.preventDefault();
+          event.stopPropagation();
+          if(opts.undoable && !ur)
+            ur = cy.undoRedo({
+              defaultActions: false
+            });
+          if(api.isCollapsible(node))
+            if (opts.undoable){
+              ur.do("collapse", {
+                nodes: node,
+                options: opts
+              });
             }
+            else
+              api.collapse(node, opts);
+          else if(api.isExpandable(node))
+            if (opts.undoable)
+              ur.do("expand", {
+                nodes: node,
+                options: opts
+              });
+            else
+              api.expand(node, opts);
+          if (options().appearOnGroupSelect) {
+            // mouse won't be in collapsed group, so shouldn't show cue
+            if (hoveredGroup && api.isExpandable(node)) {
+              hoveredGroup = null;
+            }
+            refreshCanvasImages();
           }
-        });
+        }
+
+        function isClickOnAnyCueRegion(event) {
+          return (hoveredGroup && isEventOnCueRegion(hoveredGroup, event)) || (selectedGroups.length > 0 && getGroupOfTappedOnCue(event));
+        }
+
+        function interceptEventWithinCue(event) {
+          if (isClickOnAnyCueRegion(event)){
+            event.stopPropagation();
```

Comment:
You had to add `preventDefault()` above for the click/touchEnd. Is it necessary for the other events as well?

Reply:
Doesn't seem to be. Probably it's the default behaviour of the click/touchend that was causing the problems.

Reply:
It is really only the touchend that was triggering another undesired default behaviour.